docker/eclair/Dockerfile

```diff
@@ -98,7 +96,14 @@ ENV ECLAIR_DATADIR=/home/eclair/
 RUN chmod -R a+x eclair-node/*
 RUN ls -al eclair-node/bin
 
+RUN curl -SLO https://raw.githubusercontent.com/ACINQ/eclair/master/contrib/eclair-cli.bash-completion \
+  && mkdir /etc/bash_completion.d \
+  && mv eclair-cli.bash-completion /etc/bash_completion.d/ \
+  && curl -SLO https://raw.githubusercontent.com/scop/bash-completion/master/bash_completion \
+  && mv bash_completion /usr/share/bash-completion/
+
 COPY docker-entrypoint.sh /entrypoint.sh
+COPY bashrc /home/eclair/.bashrc
 
 RUN chmod a+x /entrypoint.sh
```

Comment:
There is an issue with Eclair nodes. After opening a channel and mining blocks, the channel state remains at `WAIT_FOR_FUNDING_CONFIRMED` for a long time. This prevents payments from being routed over the new channel immediately. As of Eclair v0.6.2, we should use the `zmqpubhashblock` bitcoind config option. Check out the [release notes](https://github.com/ACINQ/eclair/blob/master/docs/release-notes/eclair-v0.6.2.md). 

I was able to resolve the issue in my branch by updating two lines in the [constants.ts](https://github.com/jamaljsr/polar/commit/2128993d6558c0c930ceec28ecb98d4b96d3a8c7#diff-e8d0a358ae2fb090c56f00a058050983cc9691b7d3622ddd443170017123e7d9L157) file. Can you bring these changes over to this PR?